### PR TITLE
Update egg-a-i-o.json

### PR DESCRIPTION
**Pull request by James-Discord**

### CODE REVIEW

**Comment by itzJOHv** [comment]
edit name change to `All In One`?

**Review by vinc3nz — CHANGES_REQUESTED** [review]
Dirt. Does nothing and there now is a mistake. It's not `your the server`

File: egg-a-i-o.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
         "version": "PTDL_v1"
     },
     "exported_at": "2020-11-16T00:26:44-05:00",
-    "name": "AIO",
+    "name": "AIO (ALL IN ONE)",
```
Comment (djohts) [inline egg-a-i-o.json:7]:
```diff
-    "name": "AIO (ALL IN ONE)",
+    "name": "AIO (All In One)",
```

File: egg-a-i-o.json [hunk 2]
```diff
@@ -26,12 +26,12 @@
     "variables": [
         {
             "name": "Startup Command",
-            "description": "The startup command for your bot to use",
+            "description": "The startup command for your the server to use",
```
Comment (djohts) [inline egg-a-i-o.json:29]:
```diff
-            "description": "The startup command for your the server to use",
+            "description": "The startup command for your server to use",
```

**Review by djohts — CHANGES_REQUESTED** [review]

**Pull request closed without merging**